src/Athens-Cairo-Tests/AthensCairoSVGSurfaceExportTest.class.st

```diff
@@ -0,0 +1,10 @@
+Class {
+	#name : #AthensCairoSVGSurfaceExportTest,
+	#superclass : #AthensCairoSurfaceExportTest,
+	#category : #'Athens-Cairo-Tests-Core'
+}
+
+{ #category : #tests }
+AthensCairoSVGSurfaceExportTest >> athensSurfaceClass [
+	^ AthensCairoPDFSurface
```

Comment:
is this potentially a typo? (as opposed to AthensCairoSVGSurface, if that exists?)